include/nbl/builtin/glsl/math/complex.glsl

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_MATH_COMPLEX_INCLUDED_
+#define _NBL_MATH_COMPLEX_INCLUDED_
+
+#include <nbl/builtin/glsl/math/constants.glsl>
+#include <nbl/builtin/glsl/math/functions.glsl>
+
+vec2 nbl_glsl_eITheta(in float _theta)
+{
+    // Use sincos from math/functions.glsl?
```

Comment:
probably not because `nbl_glsl_sincos` is has a comment "valid only for `theta` in [-PI,PI]"
(not sure if thats a problem anymore, but lets play it safe)